src/Console/Util/PhpProcess.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Copyright © 2017-2018 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Infection\Console\Util;
+
+use Composer\XdebugHandler\XdebugHandler;
+use Symfony\Component\Process\Process;
+
+/**
+ * Vanilla PHP process and utility functions
+ *
+ * @internal
+ */
+final class PhpProcess extends Process
+{
+    /**
+     * @var string|bool
+     */
+    private static $phprc;
+
+    /**
+     * Setups a default Xdebug-free environment for all subprocesses:
+     *
+     * - PHPRC should point to our temporary php.ini (we need to save a previous value)
+     *
+     * - PHP_INI_SCAN_DIR should be made blank because our php.ini has all we need
+     *   (a previous value can be found in XdebugHandler::getRestartSettings()["scanDir"])
+     */
+    public static function setupEnvironment()
```

Comment:
I suggest to rename it to smth like `setupXdebugFreeSubprocessesEnvironment()` or `setupXdebugFreeEnvironmentForSubprocesses()` or at least `setupXdebugFreeEnvironment()`.

It confused me each time I saw this method call because of the too general name.

What do you think?

Reply:
That's good to go.